docs/operation/bootstrap_backup_restore.rst

```diff
@@ -0,0 +1,49 @@
+Bootstrap node backup restore
+=============================
+This section describes a reliable way for backup and restore a **MetalK8s**
+bootstrap node.
+
+Backup
+******
+
+A backup file is generated at the end of the bootstrap.
+
+To create a new backup file you just have to run the backup script
+
+.. code::
+
+    /srv/scality/metalk8s-X.X.X/backup.sh
+
+Backup archive are stored in `/var/lib/metalk8s/` directory
+
+Restore
+*******
+
+.. warning::
+
+    You can't use the restore script if you not have HA apiserver as SSH
+    informations to reconfigure all nodes are stored into apiserver.
```

Comment:
Replace `as SSH informations to reconfigure all nodes are stored into apiserver` by `because the some information required to reconfigure the others nodes are stored in the apiserver`